config/software/python-nghttp2.rb

```diff
@@ -0,0 +1,10 @@
+name "python-nghttp2"
+default_version "2.10.3"
+
+dependency "python"
+dependency "pip"
+
+build do
+  ship_license "https://raw.githubusercontent.com/andymccurdy/redis-py/master/LICENSE"
```

Comment:
I think we should use https://github.com/nghttp2/nghttp2/blob/master/COPYING as the license here